content/en/docs/installation/kubernetes.md

```diff
@@ -1,342 +0,0 @@
----
```

Comment:
Should these have been deleted in #647  ?

Reply:
Yes, and they were; I asked @maelvls to rebase and that might have added them again.
Not sure why there was a merge conflict...

Reply:
Arghh, I'll look into it. That might be due to the merge I did ( ab959b5d085ee05f91f42dc509b138d3a084dd69).

Reply:
I mistakenly resolved the conflict between ada4f5ab02171765bf493037741965d1ed6c78a1 and 3186145ecaf410b6536f77115286cc507fc65a01 by choosing 3186145ecaf410b6536f77115286cc507fc65a01... my fault

Really anoying that Git does not record any conflict resolution information. Things like `git log -p --cc ab959b5` are useless when looking for why a file has disappeared/appeared after a conflict...